uint64 timestamp    # in microseconds since system start

float32 roll	    # body angular rates in NED frame
float32 pitch	    # body angular rates in NED frame
float32 yaw	    # body angular rates in NED frame
float32 thrust	    # thrust normalized to 0..1
